### ISSUE TITLE
Translation editing and live reload

### ISSUE THREAD

**Issue by kellpossible** [kellpossible/cargo-i18n#83]
+ It would be very nice to have a library that can interface with this system and provide translation editing capabilities in the form of a web interface which can be optionally embedded with a user's application in support of translation efforts (they can distribute a special build to translators).
+ It would also be nice to have some support in `i18n-embed` for a live reload from file system for people editing files by hand. Perhaps using [notify](https://github.com/notify-rs/notify).

**Comment by kellpossible** [comment]
While working on https://github.com/kellpossible/avalanche-report , it's very annoying to edit or update translations, it requires a full recompile. When working with localizations in dynamic templates which support live reload (via https://docs.rs/minijinja-autoreload/latest/minijinja_autoreload/ ), it is especially apparent that this feature would be very nice to have to completely avoid a rust compile cycle.

**Comment by wiiznokes** [comment]
I don't know the internal of this crate, but could it be possible to add a `build.rs` file to rebuild what is necessary for the `fl!` macro to work ? (https://users.rust-lang.org/t/equivalent-of-cargo-rerun-if-changed-for-procedural-macros/37036)

**Comment by kellpossible** [comment]
@wiiznokes sorry I don't quite understand, is this relating to live reload capability (having translations reload while your application is running)?

**Comment by wiiznokes** [comment]
No, it's just for rust analyzer. It is currently broken if you add a translation and you use the fl macro

**Comment by kellpossible** [comment]
Thanks @wiiznokes sounds like a new bug report!
